frontend/src/components/UserSignUp/EventCard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+// Displays volunteer event information.
+
+export default function EventCard({ event }) {
+  const EventCardWrapper = styled.div`
+    position: absolute;
+    width: 35%;
+    height: 30%;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+    background: #c1d741;
+    border-radius: 30px;
+    padding: 2%;
+  `;
+
+  const Header = styled.h1`
+    display: flex;
+    flex-direction: row;
+    position: static;
+    left: 0%;
+    top: 0%;
+    font-family: Urbanist, default;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 100%;
+    line-height: 43px;
+    color: #003c45;
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+  `;
+
+  const Name = styled.h1`
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+    font-size: 36px;
+  `;
+
+  const Slots = styled.h1`
+    padding: 0%;
+    margin: 0%;
+    border: 0%;
+    padding-left: 30%;
+    font-size: 36px;
```

Comment:
Maybe shrink the font size for slots a bit? The name and slots should both be able to fit in one line.